src/index.html

```diff
@@ -22,6 +22,436 @@
+          </h1>
+          <h2 class="card__code">Product code: 195434</h2>
+        </div>
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star stars__star--1"></div>
+            <div class="stars__star stars__star--2"></div>
+            <div class="stars__star stars__star--3"></div>
+            <div class="stars__star stars__star--4"></div>
+            <div class="stars__star stars__star--5"></div>
+          </div>
+          <p class="reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price-container">
+          <h2 class="card__price">Price:</h2>
+          <p class="card__price-number">$2,199</p>
+        </div>
+
+        <div class="card__button">
+          <a
+            href="#"
+            class="card__buy-link"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card__image-container">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="product image"
+          />
+        </div>
+
+        <div class="card__content">
+          <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+          <h2 class="card__code">Product code: 195434</h2>
+        </div>
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star stars__star--1"></div>
+            <div class="stars__star stars__star--2"></div>
+            <div class="stars__star stars__star--3"></div>
+            <div class="stars__star stars__star--4"></div>
+            <div class="stars__star stars__star--5"></div>
+          </div>
+          <p class="reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price-container">
+          <h2 class="card__price">Price:</h2>
+          <p class="card__price-number">$2,199</p>
+        </div>
+
+        <div class="card__button">
+          <a
+            href="#"
+            class="card__buy-link"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card__image-container">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="product image"
+          />
+        </div>
+
+        <div class="card__content">
+          <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+          <h2 class="card__code">Product code: 195434</h2>
+        </div>
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star stars__star--1"></div>
+            <div class="stars__star stars__star--2"></div>
+            <div class="stars__star stars__star--3"></div>
+            <div class="stars__star stars__star--4"></div>
+            <div class="stars__star stars__star--5"></div>
+          </div>
```

Comment:
Fix classes everywhere
```suggestion
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
```